og_migrate/og_migrate.module

```diff
@@ -30,10 +30,23 @@ function og_migrate_migrate_prepare_row(Row $row, MigrateSourceInterface $source
     }
   }
 
-  if (in_array($plugin_id, ['d7_field_instance', 'd7_field_instance_per_view_mode', 'd7_field_instance_per_form_display'])) {
-    // The "og_membership_type_default" bundle is now "default".
-    if ($row->getSourceProperty('bundle') == 'og_membership_type_default') {
-      $row->setSourceProperty('bundle', 'default');
+  $entity_type = $row->getSourceProperty('entity_type');
+  $bundle = $row->getSourceProperty('bundle');
+
+  // The "og_membership_type_default" bundle is now "default".
+  $old_default_bundle = 'og_membership_type_default';
+  $new_default_bundle = 'default';
+
+  if ($entity_type == 'og_membership' && $bundle == $old_default_bundle) {
+    if ($plugin_id == 'd7_field_instance' && $row->getSourceProperty('field_name') == 'og_membership_request') {
+      $fields = \Drupal::service('entity_field.manager')->getFieldDefinitions($entity_type, $new_default_bundle);
+      if(isset($fields['og_membership_request'])){
```

Comment:
Small coding standard fix: There should be a space between control operators and the opening parentheses.